lib/modules/platform/types.ts

```diff
@@ -23,6 +23,7 @@ export interface PlatformResult {
   renovateUsername?: string;
   token?: string;
   gitAuthor?: string;
+  hostRules?: HostRule[]; // return these only if _additional_ rules/hosts are required
```

Comment:
```suggestion
  /*
   * return these only if _additional_ rules/hosts are required
   */
  hostRules?: HostRule[];
```
use tsdoc comment style